src/client/Microsoft.Identity.Client/TokenCache.Notifications.cs

```diff
@@ -41,28 +41,31 @@ bool ITokenCacheInternal.IsExternalSerializationConfiguredByUser()
                 (this as ITokenCacheInternal).IsAppSubscribedToSerializationEvents();
         }
 
-        async Task ITokenCacheInternal.OnAfterAccessAsync(TokenCacheNotificationArgs args)
+        Task ITokenCacheInternal.OnAfterAccessAsync(TokenCacheNotificationArgs args)
         {
             AfterAccess?.Invoke(args);
```

Comment:
`AfterAccess` calls a user defined callback, which can error out. I am not sure it's worth testing all implications for this potential perf gain.